### PR TITLE
fix TypeError

### PR DESCRIPTION
**Pull request by anis3**

### CODE REVIEW

File: lib/resolveurl/plugins/youtube.py [hunk 1]
```diff
@@ -36,3 +36,6 @@ def get_settings_xml(cls):
         xml = super(cls, cls).get_settings_xml()
         xml.append('<setting label="This plugin calls the youtube addon -change settings there." type="lsep" />')
         return xml
+
+    def get_url(self, host, media_id):
```
Comment (Gujal00) [inline lib/resolveurl/plugins/youtube.py:40]: Why is this required? there are no calls for get_url in the plugin

**Comment by anis3** [comment]
play a youtube video with smr link tester


Starting Kodi (18.9 (18.9.0) Git:20201023-0655c2c718). Platform: Windows NT x86 64-bit


Error Type: <type 'exceptions.TypeError'>
Error Contents: Can't instantiate abstract class YoutubeResolver with abstract methods get_url
Traceback (most recent call last):
File "C:\Users\xxx\AppData\Roaming\Kodi\addons\plugin.video.smr_link_tester\default.py", line 286, in <module>
sys.exit(main())
File "C:\Users\xxx\AppData\Roaming\Kodi\addons\plugin.video.smr_link_tester\default.py", line 282, in main
url_dispatcher.dispatch(mode, queries)
File "C:\Users\xxx\AppData\Roaming\Kodi\addons\plugin.video.smr_link_tester\resources\lib\url_dispatcher.py", line 95, in dispatch
self.func_registry[mode](*args, **kwargs)
File "C:\Users\xxx\AppData\Roaming\Kodi\addons\plugin.video.smr_link_tester\default.py", line 194, in play_link
hmf = resolveurl.HostedMediaFile(url=link)
File "C:\Users\xxx\AppData\Roaming\Kodi\addons\script.module.resolveurl\lib\resolveurl\hmf.py", line 85, in __init__
self.__resolvers = self.__get_resolvers(include_disabled, include_universal, include_popups)
File "C:\Users\xxx\AppData\Roaming\Kodi\addons\script.module.resolveurl\lib\resolveurl\hmf.py", line 113, in __get_resolvers
resolver_cache[klass] = klass()
TypeError: Can't instantiate abstract class YoutubeResolver with abstract methods get_url
-->End of Python script error report<--

**Comment by Gujal00** [comment]
Seems to happen only on Kodi 18, no idea why. Works fine on Kodi19 without this change.
